src/stavenote.ts

```diff
@@ -259,6 +259,12 @@ export class StaveNote extends StemmableNote {
               // shift lower voice right
               noteL.note.setXShift(xShift);
             }
+          } else if (noteU.note.hasStem()) {
+            noteU.stemDirection = -noteU.note.getStemDirection();
+            noteU.note.setStemDirection(noteU.stemDirection);
```

Comment:
Thanks Rodrigo. We should only change the stem direction in controlled situations. Can we move this into the `if (voices != 2)` branch? (otherwise we'll have weird issues with  stems changing direction in other situations.)

Reply:
Hi Mohit, this is already, I believe, in a very controlled situation. All these conditions have to be fulfilled:
- there are two voices
- only one of the notes has stem (the other one is a whole)
- they are not in the same line
- they overlap (the stem overlaps with the whole)

There are no other visual differences in all the tests (this is the only one where the change of stem direction was applied)

Reply:
Thanks Rodrigo -- I didn't realize that this was inside the `voices === 2` branch.